scripts/cut_release_branch.py

```diff
@@ -75,9 +77,7 @@ def _verify_local_repo_is_clean():
     """Checks that the local Git repo is clean."""
     git_status_output = subprocess.check_output(
         ['git', 'status']).strip().split('\n')
-    branch_is_clean = (
-        git_status_output[1] == 'nothing to commit, working directory clean')
-    if len(git_status_output) > 2 or not branch_is_clean:
+    if not BRANCH_IS_CLEAN_MESSAGE in git_status_output:
```

Comment:
Sorry about this -- I found out when trying to do the release that on some branches there's an extra line in "git status" saying something about the branch being up-to-date. Hence we can't always assume that the message we're checking for always appears on the second line.